openbb_terminal/cryptocurrency/onchain/topledger_model.py

```diff
@@ -0,0 +1,102 @@
+"""Topledger model"""
+__docformat__ = "numpy"
+
+import logging
+from typing import Any, Optional, Tuple
+
+import pandas as pd
+
+from openbb_terminal.decorators import log_start_end
+from openbb_terminal.helper_funcs import request
+from openbb_terminal.rich_config import console
+
+logger = logging.getLogger(__name__)
+
+MAPPING = {
+    "tensor": {
+        "api_key": "8eSjJnSuOC7uLWkrTQoWADStkNS2ZIy94pH5CsNn",
+        "queries": [
```

Comment:
nice, we can also use this array to create the choices i mentioned two comments above on the controller 👍